imap_processing/codice/codice_l0.py

```diff
@@ -42,7 +32,14 @@ def decom_packets(packet_file: Path) -> list:
     list : list
         all the unpacked data.
     """
+    descriptor = (
```

Comment:
There are a lot of split/replace going on here. Could you give a quick example of input -> output of what these transforms are doing.